source/runbooks/disabling-network-connections-if-needed.html.md.erb

```diff
@@ -0,0 +1,106 @@
+---
+owner_slack: "#modernisation-platform"
+title: Disabling network access if required
+last_reviewed_on: 2024-02-14
+review_in: 6 months or when is resolved.
+---
+
+# <%= current_page.data.title %>
+
+## Overview
+
+There may be occassions where there is a network attack on modernisation platform environments that could require a way of stopping access. This document lists teo possible solutions to the issue.
+
+[NOTE]
+
+> All the steps below assume you can access the system through the AWS console and do not require changes to the terraform code.
+
+## Establish if there is an attack or unwanted access
+
+Establish if there is unwanted access or an attack taking place by checking the details on cloudwatch in both core-network-services and core-logging. It should be obvious from the contents of the logs.
+Sample examples are shown below the text.
+
+On core-network-services examine the contents, for production services, of
+
+- fw-live-data-inline-inspection-logs-ulpiqfaq
+- live\*data-vpc-flow-logs-qoa52v4z
+- external-inspection-flow-logs
+
+or the following for non-production services
+
+- non_live_data-vpc-flow-logs-fro8eqat
+- fw-non-live-data-inline-inspection-logs-ks3lnpsk
+
+On core-logging examine the contents, for production services, of
+
+- live_data-vpc-flow-logs
+  or the following for non-production services
+- non_live_data-vpc-flow-logs
+
+### Sample screenshots
+
+[core-network-services sample](../images/core-network-services.png)
+
+[core-logging sample](../images/core-logging.png)
+
+## If an issue has been established
+
+You need to then logon to the environment for which there is an issue
+
+1. Connect to the service involved.
+2. INITIALLY detatch the environment from the VPC.
+
+[CAUTION]
+
+> **NOTE THIS IS DRASTIC AND LIKELY TO CAUSE OTHER issues.**
```

Comment:
what other issues? the impact/risk needs documenting in more details

Reply:
Made a note. It will stop all applications using the VPC from accessing the platform. This is because we have VPCs that just attach to the like of hmpss so everything on hmpps will become inaccessible.